packages/dev-utils/src/ganache-test.ts

```diff
@@ -95,7 +95,7 @@ export async function getContractFromEvent(
   let currBlock: any
   let contractAddress: any
   const target = web3.utils.sha3(eventSignature)
-  let matchesFound = 0
```

Comment:
This is just reverting changes I made to make tests pass with the old ReleaseGold initialization mechanism